### PR TITLE
Move away from kubernetes

### PR DESCRIPTION
**Pull request by enolfc**
Try a simple approach with cron configured with ansible.
Not much different from the k8s approach but for sure will not require a k8s master that takes some resources. Although simpler, it may be harder to scale if this grows too much

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,67 @@
 # fedcloud-catchall-operations
 
-Operation of fedcloud integration components for selected providers. 
-This is a helm chart that will deploy the catchall operations for
-the EGI federated cloud.
+Operation of fedcloud integration components for selected providers.
+This is a set of docker containers and an ansible role to operate the
+federation components of the EGI Cloud
```
Comment (gwarf) [inline README.md:5]: EGI Cloud Compute?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,67 @@
...
 
 ## What this does?
```
Comment (gwarf) [inline README.md:7]: Punctuation in title may not be ideal, it's often discouraged (both by the linters and also in some guidelines)

File: README.md [hunk 3]
```diff
@@ -1,75 +1,67 @@
...
 
-The chart will create:
-- a cron job per site that will execute the cloud-info-provider for every 
+The ansible will create:
```
Comment (gwarf) [inline README.md:9]:
```suggestion
The ansible playbook will create:
```

File: README.md [hunk 4]
```diff
@@ -1,75 +1,67 @@
...
+- a configuration directory `/etc/fedcloud/vos/<vo>` for every VO
+  that will contain credentials to authenticate into that VO
```
Comment (gwarf) [inline README.md:11]:
```suggestion
  that will contain credentials to authenticate with that VO
```

Not sure about the proper wording, but into doesn't look great

File: README.md [hunk 5]
```diff
@@ -1,75 +1,67 @@
...
+- a cloud-info-provider configuration per site that takes
+  credential info from `/etc/fedcloud/vos/<vo>` and sends information
+  to the AMS queue
+- a cron job per site that will execute the cloud-info-provider for every
   vo supporteb by the site and push the information to the AMS in order
```
Comment (gwarf) [inline README.md:16]:
```suggestion
  vo supported by the site and push the information to the AMS in order
```

File: README.md [hunk 6]
```diff
@@ -1,75 +1,67 @@
...
-- a service per site and VO that will execute cloudkeeper-os to synchronise
-  images pushed by cloudkeeper cron.
-- a cron job per site and VO that executes cloudkeeper with the cloudkeeper-os
-  service as backend to fetch the VO-wide image list from AppDB.
 
-## Installing the chart
+## Deploying the chart
 
-helm install -f sites.yaml -f secrets.yaml --name fedcloud fedcloud-ops
+```
```
Comment (gwarf) [inline README.md:21]:
```suggestion
```sh
```

File: README.md [hunk 7]
```diff
@@ -1,75 +1,67 @@
...
+- `playbook.yaml` is an ansible playbook that just uses the `fedcloud-catchall-ops`
+  role to configure the host
 
+### Configuration
 
-### Format of `sites`
+The `sites` directory contains a YAML file per site to configure following this
+format:
 
 ```
```
Comment (gwarf) [inline README.md:38]:
```suggestion
```yaml
```

File: README.md [hunk 8]
```diff
@@ -1,75 +1,67 @@
...
+       project_id: <project id supporting the VO vo name at the site>
+     # any other optional configuration for cloud-info-provider, e.g:
+     defaultNetwork: private | public | private_only | public_only
+     publicNetwork: <name of the public network>
 ```
+
+The role expects the following variables to be defined:
+
+- `vos` a map that contains entry for each VO with the Check-in credentials:
+  ```
```
Comment (gwarf) [inline README.md:55]:
```suggestion
  ```yaml
```

File: roles/fedcloud-catchall-ops/tasks/cloud-info.yaml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    job: >
+      flock -n -w 250 /var/lock/cloud-info/{{ site.gocdb }} 
+      docker run -v /etc/fedcloud:/etc/fedcloud:ro
+      --env-file /etc/fedcloud/cloud-info/{{ site.gocdb }}.env
+      {{ cloud_info_image }} >> /var/log/cloud-info/{{ site.gocdb }}.log 2>&1
+    cron_file: "cloud-info-{{ site.gocdb }}"
+  with_fileglob:
+  - "sites/*.yaml"
+
+
```
Comment (gwarf) [inline roles/fedcloud-catchall-ops/tasks/cloud-info.yaml:52]:
```suggestion
```

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
Can't say much on the content itself, I trust it will work or be fixed :)
Added a few cosmetic remarks.

**Comment by enolfc** [comment]
I have fixed the README.md and updated sites configuration. 
As I get the service account enrolled into more VOs these will be further populated, but that will come in other PRs

**Review by gwarf — APPROVED** [review]
LGTM!

**Pull request merged**